api/samples/cbr.c

```diff
@@ -232,10 +232,9 @@ insert(hash_table_t table, app_pc addr, cbr_state_t state)
 static void
 at_taken(app_pc src, app_pc targ)
 {
-    dr_mcontext_t mcontext = {
-        sizeof(mcontext),
-        DR_MC_ALL,
-    };
+    dr_mcontext_t mcontext;
```

Comment:
optional: Do you think a reader might wonder why we're doing it this way?  If it were a small struct using {} is simpler.  We've found that the time taken to zero the large struct shows up as noticeable overhead for optimized clients.  Maybe a simple comment to make that clear?

Reply:
Done. Added the comment in just `api/samples/cbr.c`, not in the (many) tests.